bigbluebutton-html5/imports/ui/components/presentation/presentation-toolbar/component.jsx

```diff
@@ -105,9 +105,15 @@ class PresentationToolbar extends PureComponent {
       if ([KEY_CODES.ARROW_LEFT].includes(which)) {
         this.previousSlideHandler();
       }
+      if ([KEY_CODES.PAGE_DOWN].includes(which)) {
```

Comment:
That conditoin could be added to above conditon, some like this:
`if ([KEY_CODES.ARROW_LEFT, KEY_CODES.PAGE_DOWN].includes(which)) {`

Reply:
I think a switch statement would look better visually.
```
switch (which) {
  case KEYCODES.ARROW_LEFT:
  case KEY_CODES.PAGE_DOWN:
    this.previousSlideHandler();
    break;
  case KEYCODES.ARROW_RIGHT:
  case KEY_CODES.PAGE_UP:
    this.nextSlideHandler();
    break;
}
```

Reply:
I agree with both of you. I'm not really a good developer, so whatever fits the most into the BBB code style, is fine by me.
@Tainan404  suggestion is also used at https://github.com/bigbluebutton/bigbluebutton/blob/add37bfda4f3727663cabd797e108c6bccf0e9de/bigbluebutton-html5/imports/ui/components/dropdown/trigger/component.jsx#L27

And @capilkey suggestion definitely looks better visually.

Reply:
Use the switch please. It'll be cleaner to maintain and see blames for it.